test/tests/linearfvkernels/advection/tests

```diff
@@ -0,0 +1,61 @@
+[Tests]
+  design = 'LinearFVAdvection.md LinearFVOutflowBC.md'
+  issues = '#25722'
+  [mms-1d-upwind]
+    type = PythonUnitTest
+    input = 'test.py'
+    test_case = TestAdvection1DUpwind
+    requirement = 'The system shall display first-order convergence for advection problems with upwind interpolation solved using a linear finite volume system on a one-dimensional domain.'
+    required_python_packages = 'pandas matplotlib'
+    installation_type = in_tree # see #26480
+  []
+  [mms-1d-linear]
+    type = PythonUnitTest
+    input = 'test.py'
+    test_case = TestAdvection1DLinear
+    requirement = 'The system shall display second-order convergence for advection problems with linear interpolation solved using a linear finite volume system on a one-dimensional domain.'
+    required_python_packages = 'pandas matplotlib'
+    installation_type = in_tree # see #26480
+  []
+  [mms-2d-upwind]
+    type = PythonUnitTest
+    input = 'test.py'
+    test_case = TestAdvection2DUpwind
+    requirement = 'The system shall display first-order convergence for advection problems with upwind interpolation solved using a linear finite volume system on a two-dimensional orthogonal mesh.'
+    required_python_packages = 'pandas matplotlib'
+    installation_type = in_tree # see #26480
+  []
+  [mms-2d-linear]
+    type = PythonUnitTest
+    input = 'test.py'
+    test_case = TestAdvection2DLinear
+    requirement = 'The system shall display second-order convergence for advection problems with linear interpolation solved using a linear finite volume system on a two-dimensional orthogonal mesh.'
+    required_python_packages = 'pandas matplotlib'
+    installation_type = in_tree # see #26480
+  []
+  [mms-2d-upwind-tris]
+    type = PythonUnitTest
+    input = 'test.py'
+    test_case = TestAdvection2DUpwindTris
+    requirement = 'The system shall display first-order convergence for advection problems with upwind interpolation solved using a linear finite volume system on a two-dimensional nonorthogonal mesh.'
+    required_python_packages = 'pandas matplotlib'
+    installation_type = in_tree # see #26480
+  []
+  [mms-2d-linear-tris]
+    type = PythonUnitTest
+    input = 'test.py'
+    test_case = TestAdvection2DLinearTris
+    requirement = 'The system shall display first-order convergence for advection problems with linear interpolation solved using a linear finite volume system on a two-dimensional nonorthogonal mesh.'
+    # This is first order in the original implementation as well. At some point somebody could take a look at the error terms.
```

Comment:
yea definitely should look into this

Reply:
IS it okay if I don't look into the error analysis in this PR?

Reply:
for sure